src/config/dynamic-theme-fixes.config

```diff
@@ -227,6 +227,11 @@ img[alt^="WEB_FreeTier"]
 .lb-is-lazyloaded
 .lb-none-v-margin.lb-img
 
+CSS
+.header-background {
+  fill: #181A1B !important;
```

Comment:
Hi, adding static colors is not good, the fix will not work in Light mode or will look strange after filter value change.

Try using `fill: ${white} !important;` or `fill: none !important;`.

Reply:
Thanks for the tip, I'm making the update and testing now.

Reply:
I have made the change requested and rebased.